modules/account-lib/src/keyPair/index.ts

```diff
@@ -1,15 +1,17 @@
 import * as coinModules from '..';
 import { BaseKeyPair } from '../coin/baseCoin';
 import { KeyPairOptions } from '../coin/baseCoin/iface';
+import { coins } from '@bitgo/statics';
+import { BuildTransactionError } from '../coin/baseCoin/errors';
 
 export function register(coinName: string, source?: KeyPairOptions): BaseKeyPair {
-  const sanitizedCoinName = coinName.trim().toLowerCase();
+  const sanitizedCoinName = coins.get(coinName.trim().toLowerCase()).family;
   const key = Object.keys(coinModules)
     .filter((k) => coinModules[k].KeyPair)
     // TODO(BG-40990): eth2 BLS keypair init error
     .find((k) => k.trim().toLowerCase() !== 'eth2' && k.trim().toLowerCase() === sanitizedCoinName);
   if (key) {
     return new coinModules[key].KeyPair(source);
   }
-  return {} as BaseKeyPair;
+  throw new BuildTransactionError(`Coin ${coinName} not supported`);
```

Comment:
maybe we can throw `NotSupported` error instead since this is generating keys instead of transaction

Reply:
Done